Extension/tslint.json

```diff
@@ -1,31 +1,122 @@
 {
     "rules": {
+        "adjacent-overload-signatures": true,
+        "align": true,
+        "array-type": [true, "array"],
+        "arrow-return-shorthand": true,
+        "ban-comma-operator": true,
+        "binary-expression-operand-order": true,
+        "callable-types": true,
+        "class-name": true,
+        "comment-format": true,
         "curly": true,
+        "encoding": true,
+        "eofline": true,
         "ext-variable-name": [
             true,
-            ["class", "pascal"],
-            ["function", "camel"]
+            [
+                "class",
+                "pascal"
+            ],
+            [
+                "function",
+                "camel"
+            ]
         ],
-        "file-header": [true, ".*"],
-        "indent": [true, "spaces", 4],
+        "file-header": [
+            true,
+            ".*"
+        ],
+        "import-spacing": true,
+        "indent": [
+            true,
+            "spaces",
+            4
+        ],
+        "label-position": true,
+        "match-default-export-name": true,
+        "member-ordering": true,
         "new-parens": true,
+        "no-arg": true,
+        "no-bitwise": true,
```

Comment:
Why no bitwise?

https://palantir.github.io/tslint/rules/no-bitwise/

Reply:
Because we don't use them and probably won't need to. More chance for errors.